### ISSUE TITLE
Remove react peer dependency

### ISSUE THREAD

**Issue by bpugh** [hodgef/simple-keyboard#3]
The package.json specifies react as a peer dependency but it's my understanding the vanillaJS version shouldn't depend on it correct?

**Comment by hodgef** [comment]
Correct! The library boilerplate uses React, but the built simple-keyboard is vanilla javascript.

**Comment by bpugh** [comment]
Oh I see. Hmm maybe there is a way we could split it out? Maybe have the demo boilerplate in a subdirectory? since if someone is adding it to an Angular project they wouldn't want to see npm warnings that they're missing react as a peer dependency.

**Comment by hodgef** [comment]
You're right, that's something that must be corrected. I can take a look into it this week. If you want to have a go at it and submit a PR, I would gladly accept it !

**Comment by hodgef** [comment]
Hello @bpugh,

This turned out to be easier than expected. Since dist does not need react, I just removed it from the peerDependencies. React will still be in the devDependencies, since it is used to build the live demo.

Here's the commit: https://github.com/hodgef/simple-keyboard/commit/958019199a4ee7cfdfdde66a865ccdad157c2187

The change is live as of ^2.3.11 and I've updated the demos.

e.g.: https://codesandbox.io/s/krzkx19rr

Regards,
Francisco Hodge
